### DISCUSSION TITLE
Log files and permissions

### DISCUSSION THREAD

**Discussion by silversword411** [amidaware/tacticalrmm#645] [General]
Someone was tracking down disk space usage on discord: https://discord.com/channels/736478043522072608/744282073870630912/866698847984418836

The access.log file is pretty huge, getting a new line every 3-6 seconds on my side. 

Question 1. Shouldn't that be included in some kind of log rotation function?

Question 2. I `mv access.log access1.log` and rebooted, and new file is created...but I'm wondering why all the different permissions on every file. Should that be looked at?

![2021-07-19_122437 - trmm log permissions](https://user-images.githubusercontent.com/13395348/126194735-1b3f1e30-8e59-4036-9645-bd30c3719ac8.png)

**Comment by bbrendon** [comment]
This issue looks to be referring to /rmm/api/tacticalrmm/tacticalrmm/private/log/access.log

I'm a believer of one log file using vhost logging style.  The distro nginx package already rotates /var/log/nginx/access.log

white may have it split for debugging or development purposes.

```
log_format vhost_combined '$server_name:$server_port $remote_addr - '
    '$remote_user [$time_local] "$request" $status $body_bytes_sent '
    '"$http_referer" "$http_user_agent"';

access_log /var/log/nginx/access.log vhost_combined;
```
